templates/trivy/trivy-sts.yaml

```diff
@@ -109,6 +109,10 @@ spec:
                 secretKeyRef:
                   name: {{ template "harbor.trivy" . }}
                   key: redisURL
+            {{- if .Values.caBundleSecretName }}
```

Comment:
The caBundleSecretName has already been implemented in trivy-sts.yaml:
https://github.com/goharbor/harbor-helm/blob/master/templates/trivy/trivy-sts.yaml#L124-L126
https://github.com/goharbor/harbor-helm/blob/master/templates/trivy/trivy-sts.yaml#L154-L156